examples/data-objects/key-value-cache/src/index.ts

```diff
@@ -32,9 +32,7 @@ import {
 import { defaultRouteRequestHandler } from "@fluidframework/aqueduct";
 import { assert } from "@fluidframework/common-utils";
 
-// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
-const pkg = require("../package.json");
-export const ComponentName = pkg.name;
+export const ComponentName = "Key-Valye-Cache";
```

Comment:
Also given that this pr involves a lot of cleanup to discourage bad practices, maybe we should change `ComponentName` here and other places touched to `DataStoreName` or similar as well?

Reply:
I did quick check and cleaned up this place (which was actually wrong) and couple other places - please see latest commit